CONTRIBUTING.md

```diff
@@ -301,10 +301,9 @@ Simply make your edits on their files and open a pull request to their branch!
 
 What happens if the continuous integration (CI) fails (for example, if the pull request notifies you that "Some checks were not successful")?
 The CI could fail for a number of reasons.
-At the bottom of the pull request, where it says whether your build passed or failed, you can click “Details” next to the test, which takes you to the Travis page.
+At the bottom of the pull request, where it says whether your build passed or failed, you can click “Details” next to the test, which takes you to a CI run log site.
 If you have the write access to the repo, you can view the log or rerun the checks by clicking the “Restart build” button in the top right.
-Please note that you need to be logged in to Travis CI with your GitHub account see the “Restart build” button.
-You can learn more about Travis in the [Continuous Integration chapter](https://the-turing-way.netlify.com/continuous_integration/continuous_integration.html) of the book!
+You can learn more about CI in the [Continuous Integration chapter](https://the-turing-way.netlify.com/continuous_integration/continuous_integration.html) of the book!
```

Comment:
```suggestion
You can learn more about CI in the {ref}`Continuous Integration chapter<rr-ci>`!
```

Reply:
Ahh, I was too quick to merge this:

I don't think it will work in this md file as this one is in fact lives outside the book content with the sole purpose for direct repo and github access (similarly to the top level README.md and other md files), and references are not resolved by github.

Reply:
Ah drats you're right! My mistake! Whoops!